src/index.js

```diff
@@ -16,4 +16,7 @@ export * from './pretty-dom'
 
 // The original name of bindElementToQueries was weird
 // The new name is better. Remove this in the next major version bump.
```

Comment:
Could you move these two comments to right above the `bindElementsToQueries` export? That's where the comment applies. We'll keep the `within` alias indefinitely.